src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -1076,6 +1082,18 @@ public Parent createParent(CmpdRegMolecule mol, Collection<BulkLoadPropertyMappi
 
 		//lookups
 		try{
+			//Currently this section of invalid values passed in will only work for string values
+			for (BulkLoadPropertyMappingDTO mapping : mappings){
+				if(mapping.getInvalidValues() != null) {
+					lookUpProperty = mapping.getDbProperty();
+					lookUpString = getStringValueFromMappings(mol, lookUpProperty, mappings);
+					invalidValues = mapping.getInvalidValues();
+					if(invalidValues.contains(lookUpString)) {
+						logger.error("Found invalid "+lookUpProperty+" with code "+lookUpString+" in list of invalid values passed to service");
+						throw new Exception("Invalid property");
```

Comment:
What information is going to be passed back to the user about this error? Will they just see "11 compounds had the error "Invalid Property" "? Or will they see more information about which invalid value caused this SDF entry to fail?

Reply:
This should restore the old behavior of ACAS which was to produce an error per invalid SDF entry. Attached are screenshots of an example of an SDF with two compounds that both had the same invalid `Lot Chemist` `"BADNAME"`

Summary:
<img width="960" alt="Screen Shot 2019-08-05 at 4 36 14 PM" src="https://user-images.githubusercontent.com/868119/62493571-76a94a80-b79f-11e9-8238-85f2efae3c92.png">

Error CSV:
<img width="710" alt="Screen Shot 2019-08-05 at 4 36 34 PM" src="https://user-images.githubusercontent.com/868119/62493662-a6585280-b79f-11e9-8c38-93f761a69f96.png">

Summary ZIP files list:
<img width="303" alt="Screen Shot 2019-08-05 at 4 40 15 PM" src="https://user-images.githubusercontent.com/868119/62493697-bec86d00-b79f-11e9-9664-b2c8bb43d423.png">

I also tested the same file with 1 good chemist and 1 bad chemist and the result is that one compound is loaded and ends up in the "registered" summary files and the other compound  errors appropriately and ends up in the "error" summary files. Screenshots:

Summary: 
<img width="912" alt="Screen Shot 2019-08-05 at 4 42 53 PM" src="https://user-images.githubusercontent.com/868119/62493988-6b0a5380-b7a0-11e9-94a3-8a8b46389773.png">


Error CSV:
<img width="867" alt="Screen Shot 2019-08-05 at 4 43 32 PM" src="https://user-images.githubusercontent.com/868119/62493946-50d07580-b7a0-11e9-8839-4fe189abe579.png">

Registered CSV:
<img width="581" alt="Screen Shot 2019-08-05 at 4 45 49 PM" src="https://user-images.githubusercontent.com/868119/62494044-82494100-b7a0-11e9-893a-f26d477accaf.png">

Reply:
Thanks, I misunderstood which try/catch block this was in. It makes sense now.